package.json

```diff
@@ -77,11 +82,31 @@
     "marked": "^0.3.6",
     "mocha": "^3.5.3",
     "prettier": "^2.1.2",
+    "rollup": "^2.33.3",
     "should": "^11.2.1",
     "webpack": "^4.44.1",
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
     "is-printable-key-event": "1.0.0"
+  },
+  "exports": {
```

Comment:
I'm having a little trouble wrapping my head around all these near-identical lines and squaring it with the documentation on [conditional exports](https://nodejs.org/api/packages.html#packages_conditional_exports). My (uninformed) spider-sense tells me we can do with less, but please correct me if I'm wrong!

Just to make sure I actually understand this: the `development` and `production` keys are to define separate package exports depending on whether `require('canvas-datagrid/development')` is called, or `require('canvas-datagrid/production')` (or the `import` equivalent) ? Do we need to explicitly define a `production` key — as in, is that not the default, on lines 106-110?

Reply:
When writing that I was referencing the Webpack documentation on the subject. I don't think they create separate exports since they don't include path characters, downstream Webpack probably uses it for extended conditionals, but yes I think they're probably redundant. Except I think I'd move the development down so if someone's using webpack with script or require and are running in development mode, the `.debug.js` will be used.